src/test/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilderTest.java

```diff
@@ -37,89 +36,139 @@
-        Mockito.when(mockReactionInst.getAttributeValue(ReactomeJavaConstants.stableIdentifier)).thenReturn(mockStableIdentifierInst);
-        Mockito.when(mockStableIdentifierInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("1234");
+        initMocks(dummyTaxonIdentifier);
+        mockProteinToReturnNoDisqualificationsForAnnotation();
         Mockito.when(mockProteinInst.getAttributeValue(ReactomeJavaConstants.compartment)).thenReturn(null);
+
         List<String> goaLines = CellularComponentAnnotationBuilder.processCellularComponents(mockReactionInst);
 
-        assertThat(goaLines.size(), is(equalTo(1)));
-        assertThat(goaLines.get(0), is(equalTo("")));
+        assertThat(goaLines.size(), is(equalTo(0)));
     }
 
     @Test
-    public void cellularComponentExcludedMicrobialSpeciesReturnsArrayWithEmptyStringTest() throws Exception {
-        PowerMockito.mockStatic(GOAGeneratorUtilities.class);
-        mockProteinSet.add(mockProteinInst);
+    public void cellularComponentExcludedMicrobialSpeciesReturnsZeroLinesTest() throws Exception {
+        final String excludedMicrobialSpeciesTaxonIdentifier = "813";
+        final String dummyGOAccession = "0004321";
+
+        initMocks(excludedMicrobialSpeciesTaxonIdentifier);
+        mockProteinCompartmentToReturnGOAccession(dummyGOAccession);
+        mockProteinToReturnMockReferenceEntity();
+
+        Mockito.when(GOAGeneratorUtilities.isValidProtein(mockProteinInst)).thenReturn(true);
+        Mockito.when(GOAGeneratorUtilities.checkProteinForDisqualification(mockProteinInst)).thenCallRealMethod();
+        Mockito.when(GOAGeneratorUtilities.hasExcludedMicrobialSpecies(mockProteinInst)).thenCallRealMethod();
 
-        Mockito.when(GOAGeneratorUtilities.retrieveProteins(mockReactionInst)).thenReturn(mockProteinSet);
-        Mockito.when(mockProteinInst.getAttributeValue(ReactomeJavaConstants.referenceEntity)).thenReturn(mockReferenceEntityInst);
-        Mockito.when(mockProteinInst.getAttributeValue(ReactomeJavaConstants.species)).thenReturn(mockSpeciesInst);
-        Mockito.when((GOAGeneratorUtilities.isValidProtein(mockReferenceEntityInst, mockSpeciesInst))).thenReturn(true);
-        Mockito.when(((GKInstance) mockSpeciesInst.getAttributeValue(ReactomeJavaConstants.crossReference))).thenReturn(mockCrossReferenceInst);
-        Mockito.when(mockCrossReferenceInst.getAttributeValue(ReactomeJavaConstants.identifier)).thenReturn("813");
-        Mockito.when(GOAGeneratorUtilities.isExcludedMicrobialSpecies("813")).thenCallRealMethod();
         List<String> goaLines = CellularComponentAnnotationBuilder.processCellularComponents(mockReactionInst);
 
         assertThat(goaLines.size(), is(equalTo(0)));
     }
+
+    private void initMocks(String taxonIdentifier) throws Exception {
+        PowerMockito.mockStatic(GOAGeneratorUtilities.class);
+        PowerMockito.mockStatic(InstanceUtilities.class);
```

Comment:
More recent versions of Mockito are now capable of mocking static methods, so we may not need PowerMockito anymore.

Reply:
Okay, I just wanted to get the original tests back to passing.  I think a separate PR for test updates would be good.